### PR TITLE
Fix several documentation issues

### PR DESCRIPTION
**Pull request by mhoss019**

### CODE REVIEW

File: docs/configure/reports/resource-metrics.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ To use an existing Prometheus installation pass the following flag to the Helm c
 --set resourcemetrics.address="http://prometheus-server.<namespace>"
 ```
 
-Alternatively, you can click on **Prometheus** from the [Report Hub](https://insights.docs.fairwinds.com/installation/report-hub/), and then:
+Alternatively, you can click on **Prometheus** from the [Report Hub](https://insights.docs.fairwinds.com/run/agent/report-hub/), and then:
```
Comment (baderbuddy) [inline docs/configure/reports/resource-metrics.md:13]: If you make this a relative link then we have a script that will catch these mistakes. `/run/agent/report-hub/`

File: docs/configure/reports/resource-metrics.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ To use an existing Prometheus installation pass the following flag to the Helm c
...
 1. Click on the **Configure** tab
 2. Enter the service address of the existing Prometheus instance
 3. Click "Save"
-4. Be sure to [Re-install the Fairwinds Insights agent](https://insights.docs.fairwinds.com/installation/report-hub/) in your cluster
+4. Be sure to [Re-install the Fairwinds Insights agent](https://insights.docs.fairwinds.com/run/agent/installation/) in your cluster
```
Comment (baderbuddy) [inline docs/configure/reports/resource-metrics.md:17]: Same note

File: docs/configure/reports/resource-metrics.md (hunk 1 above)
Comment (rbren) [inline docs/configure/reports/resource-metrics.md:13]: 🤔 we should probably make these relative links

File: docs/run/admission/setup.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ If you don't have cert-manager, or if you'd like to provide your own certificate
 stored in your cluster.
 
 ## Installation
-To use the Admission Controller and install it on your cluster, navigate to the [Report Hub](https://insights.docs.fairwinds.com/installation/report-hub/) and select "Admission Controller". (You will need to re-install the Helm chart after selecting the Admission Controller.)
+To use the Admission Controller and install it on your cluster, navigate to the [Report Hub](https://insights.docs.fairwinds.com/run/agent/report-hub/) and select "Admission Controller". (You will need to re-install the Helm chart after selecting the Admission Controller.)
```
Comment (baderbuddy) [inline docs/run/admission/setup.md:11]: We might have been bad about fully qualified links...

Comment (rbren) [inline docs/run/admission/setup.md:11]: ditto

File: docs/technical-details/self-hosted/installation.md [hunk 4]
```diff
@@ -32,11 +32,11 @@ kubectl port-forward -n fairwinds-insights svc/fairwinds-insights-dashboard 8080
...
-* [File Storage](file-storage): set up a durable place to store files (S3 or Minio).
-* [Ingress](ingress): host Insights behind a custom domain
-* [Sessions](sessions): Generate permanent session keys in order to preserve running sessions when Insights is updated.
-* [Email](email): In order to confirm email addresses and add new users, you'll need to set up an email provider.
+* [Database](https://insights.docs.fairwinds.com/technical-details/self-hosted/database/): set up a durable Postgres database for your data
+* [File Storage](https://insights.docs.fairwinds.com/technical-details/self-hosted/file-storage/): set up a durable place to store files (S3 or Minio).
+* [Ingress](https://insights.docs.fairwinds.com/technical-details/self-hosted/ingress/): host Insights behind a custom domain
+* [Sessions](https://insights.docs.fairwinds.com/technical-details/self-hosted/sessions/): Generate permanent session keys in order to preserve running sessions when Insights is updated.
+* [Email](https://insights.docs.fairwinds.com/technical-details/self-hosted/email/): In order to confirm email addresses and add new users, you'll need to set up an email provider.
```
Comment (rbren) [inline docs/technical-details/self-hosted/installation.md:39]: ditto

File: docs/run/agent/report-hub.md [hunk 5]
```diff
@@ -30,7 +30,6 @@ You can customize any of the reports by clicking on its tile.
 Some common options are:
 * `schedule` - a Cron expression describing when to run this report
 * `timeout` - the maximum time this report should run before an error is triggered
-* `save as organization default settings` - if checked, these settings will become your organization's default settings. All new and existing clusters will inherit these settings, unless custom settings have been specified for that cluster.
```
Comment (baderbuddy) [inline docs/run/agent/report-hub.md:33]: ??? why did you remove this?

File: docs/technical-details/self-hosted/installation.md (hunk 4 above)
Comment (baderbuddy) [inline docs/technical-details/self-hosted/installation.md:39]: Let's make these relative too

**Review by rbren — COMMENTED** [review]
Thanks Mahmoud!

We have a CI check for broken links, but it only works for relative links (i.e. links that don't start with `insights.docs.fariwinds.com`) - any reason you added the domain to some of them?

**Review by baderbuddy — APPROVED** [review]
This is all good, but taking out the domain name on the links will make things a little easier for the future.

File: docs/run/agent/report-hub.md (hunk 5 above)
Comment (mhoss019) [reply docs/run/agent/report-hub.md:33]: How do I make that option appear? I'm logged in as an admin of an organization and it doesn't come up

Comment (baderbuddy) [reply docs/run/agent/report-hub.md:33]: I guess it got removed, I had no idea

**Comment by mhoss019** [comment]
Thanks for the reviews Robert and Bader! I changed the links to relative links.

File: docs/technical-details/self-hosted/installation.md [hunk 6]
```diff
@@ -32,11 +32,11 @@ kubectl port-forward -n fairwinds-insights svc/fairwinds-insights-dashboard 8080
...
 The default configuration will give you a working version of Fairwinds Insights.
 But there are a few issues you'll want to solve before starting to use it seriously:
-* [Database](database): set up a durable Postgres database for your data
-* [File Storage](file-storage): set up a durable place to store files (S3 or Minio).
-* [Ingress](ingress): host Insights behind a custom domain
-* [Sessions](sessions): Generate permanent session keys in order to preserve running sessions when Insights is updated.
-* [Email](email): In order to confirm email addresses and add new users, you'll need to set up an email provider.
+* [Database](/technical-details/self-hosted/database): set up a durable Postgres database for your data
+* [File Storage](/technical-details/self-hosted/file-storage): set up a durable place to store files (S3 or Minio).
+* [Ingress](/technical-details/self-hosted/ingress): host Insights behind a custom domain
```
Comment (rbren) [inline docs/technical-details/self-hosted/installation.md:37]: do these not work as-is?

**Review by rbren — APPROVED** [review]
